rdf_differ/config.py

```diff
@@ -14,3 +14,5 @@
 
 FUSEKI_USERNAME = os.environ.get('FUSEKI_USERNAME', 'admin')
 FUSEKI_PASSWORD = os.environ.get('FUSEKI_PASSWORD', 'admin')
+
+FLASK_SECRET_KEY = os.environ.get('SECRET_KEY', 'secret key')
```

Comment:
what is this the key for?

Reply:
[docs](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY):
> A secret key that will be used for securely signing the session cookie and can be used for any other security related needs by extensions or your application. It should be a long random string of bytes, although unicode is accepted too.